### ISSUE TITLE
Missing info in XML: location, width, valign, alt

### ISSUE THREAD

**Issue by Dschaehn** [doxygen/doxygen#8312]
**Describe the bug**
While building our custom rendering on top of DoxyGen generated XML, we found that some XML tags/attributes are missing for our usecase:
- `<location>` of pages
- width attribute in `<table>` and `<entry>`
- valign in `<entry>`
- alt in `<image>`

Since that are quite common attributes which work fine for html output, can you please add them in XML as well? We did this already locally and I will propose the change as pull request.

**Expected behavior**
The XML for the minimal sample attached below should include:

~~~
<para><table rows="1" cols="2" width="500"><row>
<entry thead="no" width="50%" valign="middle"><para></para>
</entry><entry thead="no"><para></para>
</entry></row>
</table>
</para>
<para><image type="html" name="image.png" alt="Image description" inline="yes"></image>
~~~

What we currently get:
~~~
<para><table rows="1" cols="2"><row>
<entry thead="no"><para></para>
</entry><entry thead="no"><para></para>
</entry></row>
</table>
</para>
<para><image type="html" name="image.png" inline="yes"></image>
~~~

**To Reproduce**
`doxygen Doxyfile` with [missing_xml_info.zip](https://github.com/doxygen/doxygen/files/5782158/missing_xml_info.zip)

**Version**
Tested with 1.9.0

**Comment by albert-github** [comment]
Due to the (automatic) link with the pull request this issue was automatically closed as the code was integrated in master on github.
Reopening issue, please don't close the issue as this will be done at the moment of an official release.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.9.1.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
